main.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
this really works this way? I mean, it looks like a comment to me 🤔

Reply:
Yes, this is a magical comment